tests/test_plots.py

```diff
@@ -0,0 +1,122 @@
+"""
+Testing the plot functions
+"""
+# external packages
+import pytest
+# internal packages
+import os
+from shutil import copy
+# local packages
+from simple_app.plots import *
+
+db_name = 'temp.db'
+db_cs = 'sqlite:///temp.db'
+
+
+@pytest.fixture(scope='module')
+def db():
+    if os.path.exists(db_name):
+        os.remove(db_name)
+    copy('SIMPLE.db', db_name)
+    assert os.path.exists(db_name)
+    # Connect to the new database and confirm it has the Sources table
+    db = SimpleDB(db_cs)
+    assert db
+    assert 'source' in [c.name for c in db.Sources.columns]
+    return db
+
+
+@pytest.fixture(scope='module')
+def test_all_sources(db):
+    assert db
+    assert all_sources(db_cs)
+    allresults, fullresults = all_sources(db_cs)
+    assert len(allresults) and len(fullresults)
+    assert type(allresults) == list
+    assert type(fullresults) == pd.DataFrame
+    return allresults, fullresults
+
+
+@pytest.fixture(scope='module')
+def test_all_photometry(db):
+    assert db
+    allphoto, allbands = all_photometry(db_cs)
+    assert len(allphoto) and len(allbands)
+    assert type(allphoto) == pd.DataFrame
+    assert type(allbands) == np.ndarray
+    return allphoto, allbands
+
+
+@pytest.fixture(scope='module')
+def test_all_parallaxes(db):
+    assert db
+    allplx = all_parallaxes(db_cs)
+    assert len(allplx)
+    assert type(allplx) == pd.DataFrame
+    return allplx
```

Comment:
Aren't these exactly the same as those in test_utils? If so, they shouldn't be here in test_plots